lib/widget/reusable_zoomable_widget.dart

```diff
@@ -5,36 +5,27 @@ class ZoomablePhoto extends StatelessWidget {
   // Image url for the graph
   final String imageUrl;
   final Color backgroundColor;
-  final double padding;
 
-  const ZoomablePhoto({this.imageUrl, this.backgroundColor, this.padding = 20});
+  const ZoomablePhoto({this.imageUrl, this.backgroundColor});
 
   @override
   Widget build(BuildContext context) {
-    return Padding (
-      padding: const EdgeInsets.only(top: 20.0),
-      child: Column(
-          crossAxisAlignment: CrossAxisAlignment.stretch,
-          children: <Widget>[
-            Expanded(
-              child: ClipRect(
-                child: PhotoView(
-                  backgroundDecoration: BoxDecoration(
-                    color: backgroundColor
-                  ),
-                  imageProvider: AssetImage(imageUrl),
-                  customSize: _getInitialZoomableImageSize(context, padding),
-                  basePosition: Alignment.topCenter,
-                ),
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.stretch,
+      children: <Widget>[
+        Expanded(
+          child: ClipRect(
+            child: PhotoView(
```

Comment:
```suggestion
            child: PhotoView(
               child: Padding(
                  padding: const EdgeInsets.all(16.0),
```

Reply:
Wrap with Padding to better space under AppBar.

Note: missing bracket ')' here but hard to made code changes in Suggestions on GitHub and get the brackets right. So re-check this locally first.

Reply:
The paddings around is fixed and will not disappear when the user zooms in. @lukesleeman Actually I prefer not to have a fix padding around the image. If the top position looks weird, how about put the image in the center?

Reply:
Oh yeah.Lets tweak this later.